os/drivers/task_manager/task_manager_drv.c

```diff
@@ -155,6 +155,15 @@ static int taskmgr_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 		}
 		break;
 	case TMIOC_RESTART:
+		tcb = sched_gettcb((int)arg);
+		if (tcb == NULL) {
+			tmdbg("Invalid pid\n");
+			return ERROR;
+		}
+		ret = sigprocmask(SIG_SETMASK, NULL, &tcb->sigprocmask);
```

Comment:
As `sigprocmask` is declared as `sigprocmask(int sig, FAR const struct sigaction *act, FAR struct sigaction *oact)`, how about use `NULL_SIGNAL_SET` instead of `NULL`.
`NULL_SIGNAL_SET` is defined as `((sigset_t)0x00000000)` at signal.h

Reply:
I think NULL and NULL_SIGNAL_SET is same. And in TizenRT, NULL_SIGNAL_SET is used when assign directly.
So I think NULL is better.